project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -1286,3 +1299,36 @@ function c4m_user_ckeditor_settings_alter(&$settings, $conf) {
     unset($settings['loadPlugins']['media']);
   }
 }
+
+/**
+ * Implements hook_module_implements_alter().
+ *
+ * Ensures c4m_user form_alter runs after VBO.
+ */
+function c4m_user_module_implements_alter(&$implementations, $hook) {
+  if ($hook != 'form_alter') {
+    return;
+  }
+
+  if (!isset($implementations['c4m_user'])) {
+    return;
+  }
+
+  $group = $implementations['c4m_user'];
+  unset($implementations['c4m_user']);
+  $implementations['c4m_user'] = $group;
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Updates value of submit button.
+ */
+function c4m_user_form_views_form_c4m_overview_og_members_page_alter(&$form, &$form_state, $form_id) {
+  $form['actions']['submit']['#value'] = t('Send');
+  if (isset($form['select_all_markup']['#markup'])) {
+    // VBO module doesn't provide an option to change 'Select all' labels.
```

Comment:
@ordavidil 
It's labels, as in case there're more users than defined to be displayed on page, there're second label: Select all members on all pages.